src/style.css

```diff
@@ -1,3 +1,103 @@
+:root {
+  --nav-active-and-hover-text-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+}
+
+nav ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+li {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  margin: 0 10px;
+}
+
+li:first-child {
+  margin-left: 0;
+}
+
+li:last-child {
+  margin-right: 0;
+}
```

Comment:
do not use tag selectors, better to add the `nav__item` to li tags and use `.nav__item` here